sonar-db/src/main/java/org/sonar/db/permission/ExtendedUserPermissionDto.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.db.permission;
+
+import javax.annotation.CheckForNull;
+
+/**
+ * Temporary replacement of {@link UserPermissionDto} as long as tables group_roles
+ * and user_roles do not have project UUID/user login but project ID/user ID.
+ */
+public class ExtendedUserPermissionDto {
```

Comment:
ExtendedUserPermissionDto has a null permission, whereas it's not the case with UserPermissionDto. In fact I should probably adjust the javadoc.